### ISSUE TITLE
Add Mask-RCNN model

### ISSUE THREAD

**Issue by innat** [keras-team/keras-cv#623]
**Short Description**

A two stage object detection model, strongly well-known for its detection and instance-wise simultaneous output. 

**Papers**
https://arxiv.org/abs/1703.06870

Publish date: 2017 
Cited by **20059** (until now)

**Existing Implementations**

A very popular and well-know `keras (tf 1.5x)` implementation. 

- https://github.com/matterport/Mask_RCNN (over **22K + star, 11K+ fork**).

**Other Information**

- tensorflow-model-zoo: https://github.com/tensorflow/models/tree/master/official/vision#Instance-Segmentation-Baselines

**Comment by MrinalTyagi** [comment]
Any decision to implement this? Would love to contribute the model.

**Comment by innat** [comment]
This needs to be first, I think.
https://github.com/keras-team/keras-cv/issues/669

**Comment by innat** [comment]
old model.

**Comment by sachinprasadhs** [comment]
Mask RCNN is now part of Keras CV here https://github.com/keras-team/keras-cv/tree/master/keras_cv/src/models/object_detection/mask_rcnn.
Feel free to close the issue once verified. Thanks

**Comment by innat** [comment]
> Mask RCNN is now part of Keras CV here https://github.com/keras-team/keras-cv/tree/master/keras_cv/src/models/object_detection/mask_rcnn. Feel free to close the issue once verified. Thanks

Does it actually work?

**Comment by innat** [comment]
- [ ] Complete end-to-end implementation.
- [ ] Checkpoints or pre-trained weights.
- [ ] Reproduce qualitative and quantitative results (reference from paper).
- [ ] End-to-end training / fine-tuning code example and achieve expected results with this implementation.

**Comment by innat** [comment]
> This issue is stale because it has been open for 14 days with no activity. It will be closed if no further activity occurs. Thank you.

**Comment by sachinprasadhs** [comment]
Thanks for reporting the issue! We have consolidated the development of KerasCV into the new KerasHub package, which supports image, text, and multi-modal models. Please read https://github.com/keras-team/keras-hub/issues/1831. KerasHub will support all the core functionality of KerasCV.

KerasHub can be installed with !pip install -U keras-hub. Documentation and guides are available at [keras.io/keras_hub](https://keras.io/keras_hub/).

With our focus shifted to KerasHub, we are not planning any further development or releases in KerasCV. If you encounter a KerasCV feature that is missing from KerasHub, or would like to propose an addition to the library, please file [an issue with KerasHub](https://github.com/keras-team/keras-hub/issues).

**Comment by innat** [comment]
@sachinprasadhs 
> If you encounter a KerasCV feature that is missing from KerasHub, or would like to propose an addition to the library, please file [an issue with KerasHub](https://github.com/keras-team/keras-hub/issues).

Firstly, identifying which feature is required or missing can be done effectively, either by practitioners or the Keras team. Tickets in `keras-cv` were created only when such features were **NOT** available or when a specific need was encountered. Commenting after a significant delay and requesting the recreation of the same issue is neither efficient nor appropriate. Kindly move this ticket (and any other similar tickets with the same context) from `keras-cv` to `keras-hub`.
